test/constructs/for_test.exs

```diff
@@ -55,7 +55,7 @@ defmodule Surface.Constructs.ForTest do
       quote do
         ~H"""
         <For each={{ fruit <- ["apples", "bananas", "oranges"] }}>
-          <ListProp prop="some string">The inner content</ListProp>
+          <ListProp prop="some string" />
```

Comment:
`ListProp` does not have a `default` slot so I decided to remove the inner content. An alternative could be to update the `ListProp` component and add the `default` slot.